Dockerfile.hpo

```diff
@@ -38,7 +38,7 @@ RUN microdnf install -y python3 \
 USER 1001
 
 # Install optuna to the default user
-RUN python3 -m pip install --user optuna requests scikit-optimize jsonschema
+RUN python3 -m pip install --user optuna requests scikit-optimize jsonschema click grpcio google-api-core
 
```

Comment:
@chandrams do we still need google-api-core here?

Reply:
Without installing google-api-core when I tried I got this error:
docker run -d --name hpo_docker_container -p 8085:8085 -p 50051:50051 hpo:grpc
```
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
38a0b80b9988820a0cda7cd2930041334c3da2bc2ee0f430751673488754dd01
(base) [csubrama@csubrama hpo]$ docker logs -f hpo_docker_container
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
Traceback (most recent call last):
  File "src/service.py", line 17, in <module>
    import rest_service, grpc_service
  File "/opt/app/src/grpc_service.py", line 21, in <module>
    from gRPC import hpo_pb2, hpo_pb2_grpc
  File "/opt/app/src/gRPC/hpo_pb2.py", line 5, in <module>
    from google.protobuf import descriptor as _descriptor
ModuleNotFoundError: No module named 'google'
```

Reply:
we don't need to pull in the full google-api (i.e to interact with with google services), we just need `protobuf` package (which is a dependency of google-api-core. I'll update the dockerfile

Reply:
Updated